installer/conf/omsagent.conf

```diff
@@ -84,9 +84,11 @@
   buffer_type file
   buffer_path /var/opt/microsoft/omsagent/state/out_oms_blob*.buffer
   buffer_queue_limit 10
+  buffer_queue_full_action drop_oldest_chunk
   flush_interval 20s
   retry_limit 10
   retry_wait 30s
+  max_retry_wait 9m
```

Comment:
See our OSTC docs GitHub page for clarity on how to use rebase. The general idea is:

* You make your changes in a branch
* Somebody else makes unrelated changes and commits to master, so your branch is out of date
* You switch to master branch and do `git pull` so your master is up to date, then switch back to your branch and do a `git rebase`
* This will have git reapply all of your changes on top of the then-current master branch. As long as there are no change conflicts, you're good!

If you have further questions, ask someone on your team, or swing by my office and I'd be happy to show you the process.